circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_inner.cpp

```diff
@@ -30,6 +31,8 @@ void initialise_end_values(PreviousKernelData<NT> const& previous_kernel, Kernel
     auto& end = public_inputs.end;
     const auto& start = previous_kernel.public_inputs.end;
     end.read_requests = start.read_requests;
+    end.nullified_commitments = start.nullified_commitments;
+    end.private_call_stack = start.private_call_stack;
```

Comment:
Should these have been here before!???

Reply:
@dbanks12 These two lines were moved from the common_XXX() function to the one specific to circuit inner. Namely, after having stripped nullified_commitments and private_call_stack from ordering circuit public_inputs.end, I needed to move these two lines here. You can see that in common.hpp, these two assignments were removed.